src/room_member.cpp

```diff
@@ -175,8 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    // TODO(Subv): Use client_port.
-    RakNet::SocketDescriptor socket;
+    RakNet::SocketDescriptor socket(client_port, 0);
```

Comment:
Then it would also fail with the current code.
Note that it uses `_hostAddress` (the argument) to read from but moves to `hostAddress` (no underscore) which is the member variable.